py/port_dml.py

```diff
@@ -14,9 +14,13 @@
 import traceback
 from pathlib import Path
 import ply.lex
-from simicsutils.internal import is_same_file
 from simicsutils.host import host_type
 
+def is_same_file(p1, p2):
+    return (not (os.path.exists(p1) or os.path.exists(p2))
+            or ((os.path.exists(p1) and os.path.exists(p2)
+                 and os.path.samefile(p1, p2))))
+
```

Comment:
if you look at the usage, both files are expected to be present, so better to replace usage with samefile right away instead.

Reply:
@mandolaerik Fixed